src/parser.js

```diff
@@ -30,6 +30,10 @@ const ANNOTATIONS = {
     code: 4,
     fragment: 'swagger-validation',
   },
+  UNCAUGHT_ERROR: {
```

Comment:
Remember to update the table in the READMe including the new code https://github.com/apiaryio/fury-adapter-swagger#parser-codes